pkg/executor/simple.go

```diff
@@ -1686,8 +1686,8 @@ func (e *SimpleExec) executeAlterUser(ctx context.Context, s *ast.AlterUserStmt)
 	}
 	activeRoles := e.Ctx().GetSessionVars().ActiveRoles
 	hasCreateUserPriv := checker.RequestVerification(activeRoles, "", "", "", mysql.CreateUserPriv)
-	hasSystemUserPriv := checker.RequestDynamicVerification(activeRoles, "SYSTEM_USER", false)
-	hasRestrictedUserPriv := checker.RequestDynamicVerification(activeRoles, "RESTRICTED_USER_ADMIN", false)
+	hasSystemUserPriv := checker.RequestDynamicVerification(activeRoles, []string{"SYSTEM_USER"}, false)
+	hasRestrictedUserPriv := checker.RequestDynamicVerification(activeRoles, []string{"RESTRICTED_USER_ADMIN"}, false)
```

Comment:
It seems that we can also use the old interface to check a list of privileges? like here

Reply:
Yes, it doable, but not very convenient in some cases. While except for ResourceGroup there is only 1 other cases that need to pass 2 permissions. I'm also not sure if there are other cases in the future.

Reply:
cc @tiancaiamao Do you think this modification is okay?

Reply:
Maybe change the interface is unnecessary.
We do not have much cases relying on the new interface.